lib/grape_oauth2/mixins/mongoid/client.rb

```diff
@@ -11,7 +11,7 @@ module Client
           include ::Mongoid::Timestamps
 
           has_many :access_tokens, class_name: Grape::OAuth2.config.access_token_class_name,
-                                   foreign_key: :client_id, dependent: :delete
+                                   foreign_key: :client_id, dependent: :delete_all
```

Comment:
Is it a backward compatible change with any versions of Mongoid?

Reply:
Mongoid 7.1.